exercises/practice/bottle-song/bottle-song.lua

```diff
@@ -0,0 +1,10 @@
+local BottleSong = {}
+
+--- Produces the lyrics to the song "Ten Green Bottles"
+-- @param start_bottles The initial number of bottles on the wall
+-- @param[opt=1] take_down The number of bottles that fall
```

Comment:
```suggestion
```

I think we should omit this for consistency

Reply:
Omitted